src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityItemBus.java

```diff
@@ -50,6 +58,7 @@ public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
 
     @Override
     public void update() {
+        IItemHandlerModifiable inventory = (isExportHatch ? this.getExportItems() : this.getImportItems());
```

Comment:
This can be moved to just after the check for `isAutoCollapse()` further down, so it does not need to be retrieved every tick.

Reply:
Not quite, because we need to check this `inventory` in the same if check as `isAutoCollapse`. But it can be moved behind the offsetTimer check.